### PR TITLE
migrate wiki-page to github

### PR DESCRIPTION
**Pull request by Yunir**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
+## Changelog
+
+### Version 1.19 (2019-05-31)
+
+-   [ JENKINS-42950](https://issues.jenkins-ci.org/browse/JENKINS-42950)
+    - Getting issue details... STATUS  Expanded secret masking to
```
Comment (jglick) [inline CHANGELOG.md]: The exporter is buggy; you need to manually fix up some Markdown output like this.

File: README.md [hunk 2]
```diff
@@ -1,60 +1,69 @@
...
-
+# Jenkins Credentials Binding Plugin
+Allows credentials to be bound to environment variables for use from
+miscellaneous build steps.
+
+Older versions of this plugin may not be safe to use. Please review the
+following warnings before using an older version:
+
+-   [Improper masking of secrets in rare
+    circumstances](https://jenkins.io/security/advisory/2018-02-05/)
```
Comment (jglick) [inline README.md]: Delete. These lines are injected by a Confluence macro and are redundant here; the plugin site automatically shows them. Again, buggy exporter.

File: CHANGELOG.md (hunk 1 above)
Comment (Yunir) [reply CHANGELOG.md]: My fault. Will fix it)

Comment (jglick) [reply CHANGELOG.md]: Thanks! I do not care too much about the historical changelog since I would expect to switch to Release Drafter anyway.

**Pull request merged**